packages/web/components/Player/Section/PlayerHero.tsx

```diff
@@ -181,11 +191,14 @@ export const PlayerHero: React.FC<Props> = ({
 
         {player?.profile?.emoji && (
           <PlayerHeroTile title="Favorite Emoji">
-            <Text ml={10} mt={0} fontSize={45} lineHeight={0.75}>
-              {player.profile.emoji}
-            </Text>
+            <Text fontSize="1.25rem">{player.profile.emoji}</Text>
           </PlayerHeroTile>
         )}
+        {/* player?.profile?.colorMask && (
+          <PlayerHeroTile title="Color Disposition">
+            <ColorDispositionDisplay {...{ person, personalityInfo }} />
+          </PlayerHeroTile>
+        ) */}
```

Comment:
what's up with that part? was this ever even displayed?

Reply:
ah. nice catch @vidvidvid didn't realise this got in. 
so peth asked me to completely remove a separate section for "color decomposition" and add it to player hero section itself.

but it's a slightly longer than just that as it involves supporting editing it as well in the `EditProfileForm` that dysbulic has built. 

so I just left this in so I can uncomment it later on. Can remove it if you think it's unnecessary

Reply:
np, we can leave it there. so if i understand correctly we're just removing the color composition all together until we find a way to support editing it in the edit form? could we just add it to the player hero section and add an edit button next to it? then we don't have to find a way to edit it in the `EditProfileForm`/

Reply:
I'm leaving this extra component I wrote so it can be used once we figure out how to integrate color disposition as well into the EditProfileForm. I think it should lie in the same modal because all that information is stored on Ceramic. But I might be wrong here, let me know what you think @vidvidvid 

Otherwise there are no changes, people can still add/remove ColorDecomposition as part of their profile.

Reply:
Cool, let's leave it there! :)